devito/ir/clusters/algorithms.py

```diff
@@ -35,137 +31,6 @@ def clusterize(exprs):
     return ClusterGroup(clusters)
 
 
-class Toposort(Queue):
```

Comment:
note for reviewers: topological sorting is now part of fusion (ie, performed inside `fuse()`)